src/core/redux/constants/index.js

```diff
@@ -40,7 +40,7 @@ export const IS_OWNER :'isOwner' = 'isOwner';
 export const USER_SEARCH_RESULTS :'userSearchResults' = 'userSearchResults';
 
 // TODO: does this belong here?
-export const INITIAL_SEARCH_RESULTS :Map = fromJS({ initial: true });
+export const INITIAL_SEARCH_RESULTS :Map = Map();
```

Comment:
I added this as part of the old version of the app... there was a reason for it but I suspect we don't need to worry about it now.